pkg/oam/util/helper.go

```diff
@@ -415,6 +420,23 @@ func GetGVKFromDefinition(dm discoverymapper.DiscoveryMapper, definitionRef v1al
 	return kinds[0], nil
 }
 
+// ConvertWorkloadGVK2Definition help convert a GVK to DefinitionReference
+func ConvertWorkloadGVK2Definition(def v1alpha2.WorkloadGVK) (v1alpha2.DefinitionReference, error) {
+	var reference v1alpha2.DefinitionReference
+	gv, err := schema.ParseGroupVersion(def.APIVersion)
+	if err != nil {
+		return reference, err
+	}
+	resource := strings.ToLower(def.Kind) + "s"
```

Comment:
you need to use `discoverymapper` to do this convert.

Reply:
Got it, I have fix it